omnigibson/prims/rigid_prim.py

```diff
@@ -544,3 +545,51 @@ def meta_link_sub_id(self):
         # Check using the old format.
         # TODO: Remove this after the next dataset release
         return int(m.LEGACY_META_LINK_PATTERN.fullmatch(self.name).group(3))
+
+    def check_points_in_volume(
+        self,
+        particle_positions_world,
+        use_visual_meshes=True,
+        mesh_name_prefixes=None,
+    ):
+        """
+        Args:
+            particle_positions_world (th.tensor): (N, 3) array of particle positions to check
+            use_visual_meshes (bool): Whether to use @volume_link's visual or collision meshes to generate points fcn
```

Comment:
we should also clarify here that we assume that these meshes are convex (and if not, we enforce it anyways by converting them into convex)